lara-typescript/src/section-authoring/components/authoring-section.tsx

```diff
@@ -4,9 +4,10 @@ import { GripLines } from "../../shared/components/icons/grip-lines";
 import { SectionColumn } from "./section-column";
 import { ICreatePageItem, ISection, ISectionItem, SectionColumns, SectionLayouts } from "../api/api-types";
 import { DraggableProvided } from "react-beautiful-dnd";
-import { UserInterfaceContext } from "../api/use-user-interface-context";
+import { UserInterfaceContext } from "../containers/user-interface-provider";
 
 import "./authoring-section.scss";
+import { usePageAPI } from "../hooks/use-api-provider";
```

Comment:
No biggee but we've been trying to keep the style imports as the last import in the component.